scene/gui/subviewport_container.cpp

```diff
@@ -228,13 +225,22 @@ void SubViewportContainer::unhandled_input(const Ref<InputEvent> &p_event) {
 }
 
 void SubViewportContainer::add_child_notify(Node *p_child) {
-	if (Object::cast_to<SubViewport>(p_child)) {
+	SubViewport *c = Object::cast_to<SubViewport>(p_child);
+	if (c) {
+		c->set_handle_input_locally(false); //Do not handle input locally here
```

Comment:
Please change the comment to
``` c++
c->set_handle_input_locally(false); // Do not handle input locally here.
```
according to the [comment style guide](https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#comment-style-guide)